shell/browser/api/electron_api_desktop_capturer.cc

```diff
@@ -181,6 +181,42 @@ DesktopCapturer::DesktopCapturer(v8::Isolate* isolate) {}
 
 DesktopCapturer::~DesktopCapturer() = default;
 
+DesktopCapturer::DesktopListListener::DesktopListListener(
+    OnceCallback update_callback,
+    OnceCallback failure_callback,
+    bool skip_thumbnails) {
+  update_callback_ = std::move(update_callback);
+  failure_callback_ = std::move(failure_callback);
+  have_thumbnail_ = skip_thumbnails;
+}
```

Comment:
nit: we typically use initializer lists:

```suggestion
DesktopCapturer::DesktopListListener::DesktopListListener(
    OnceCallback update_callback,
    OnceCallback failure_callback,
    bool skip_thumbnails) :  update_callback_(std::move(update_callback)), 
                                              failure_callback_(std::move(failure_callback)),                                               
                                              have_thumbnail_(skip_thumbnails) {}
```

Reply:
I've pushed new commits to address the remaining comments.